sdk/src/types/api/core/response.rs

```diff
@@ -25,10 +25,23 @@ pub struct InfoResponse {
     pub version: String,
     pub status: StatusResponse,
     pub metrics: MetricsResponse,
-    pub supported_protocol_versions: Vec<u8>,
-    pub protocol: ProtocolParameters,
+    pub protocol_parameters: ProtocolParametersResponse,
     pub base_token: BaseTokenResponse,
-    pub features: Vec<String>,
+    pub features: Box<[String]>,
+}
+
+impl InfoResponse {
+    pub fn latest_protocol_parameters(&self) -> &ProtocolParameters {
```

Comment:
Please harmonise, you have protocol in this method name but not the other 2

Reply:
done